src/components/checkout/payments/ep-stripe-payment/EpStripePayment.module.scss

```diff
@@ -0,0 +1,77 @@
+.stripe {
+  button {
```

Comment:
just curious; did this styling come from stripe?

Reply:
@samblacklock yeah, I used them from the Stripe Elements demo